bert_e/tests/test_bert_e.py

```diff
@@ -190,6 +190,66 @@ def dynamic_filtering(request):
 class QuickTest(unittest.TestCase):
     """Tests which don't need to interact with an external web services"""
 
+    def test_compare_branches_major_minor_vs_major_only(self):
+        from bert_e.workflow.gitwaterflow.branches import compare_branches
+        branch1 = ((4, 3),)
+        branch2 = ((4, None),)
+        self.assertEqual(compare_branches(branch1, branch2), -1)
+
+    def test_compare_branches_major_only_vs_major_only_returns_0(self):
+        from bert_e.workflow.gitwaterflow.branches import compare_branches
+        branch1 = ((4, None),)  # major-only
+        branch2 = ((4, None),)     # major.only
+        self.assertEqual(compare_branches(branch1, branch2), 0)
+
+    def test_compare_branches_major_only_vs_major_minor(self):
+        from bert_e.workflow.gitwaterflow.branches import compare_branches
+        branch1 = ((4, None),)
+        branch2 = ((4, 3),)
+        self.assertEqual(compare_branches(branch1, branch2), 1)
+
+    def test_compare_branches_major_minor_micro_vs_major_minor(self):
+        from bert_e.workflow.gitwaterflow.branches import compare_branches
+        branch1 = ((4, 3, 2),)
+        branch2 = ((4, 3),)
+        self.assertEqual(compare_branches(branch1, branch2), 2)
+
+    def test_compare_branches_major_minor_vs_major_minor_micro(self):
+        from bert_e.workflow.gitwaterflow.branches import compare_branches
+        branch1 = ((4, 3),)
+        branch2 = ((4, 3, 2),)
+        self.assertEqual(compare_branches(branch1, branch2), -2)
+
+    def test_master_queue_missing_major_only(self):
+        # Covers version with length 1
+        from bert_e.exceptions import MasterQueueMissing
+        exc = MasterQueueMissing((10,))
+        self.assertIn('q/10 is missing', str(exc))
+
+    def test_master_queue_missing_major_minor(self):
+        # Covers version with length 2
+        from bert_e.exceptions import MasterQueueMissing
+        exc = MasterQueueMissing((10, 0))
+        self.assertIn('q/10.0 is missing', str(exc))
+
+    def test_master_queue_missing_major_minor_none_micro(self):
+        # Covers version with length >=3 and micro is None
+        from bert_e.exceptions import MasterQueueMissing
+        exc = MasterQueueMissing((10, 0, None))
+        self.assertIn('q/10.0 is missing', str(exc))
+
+    def test_master_queue_missing_major_minor_micro(self):
+        # Covers version with length >=3 and micro is not None
+        from bert_e.exceptions import MasterQueueMissing
+        exc = MasterQueueMissing((10, 0, 5))
+        self.assertIn('q/10.0.5 is missing', str(exc))
+
+    def test_master_queue_missing_fallback(self):
+        # Covers fallback case
+        from bert_e.exceptions import MasterQueueMissing
+        exc = MasterQueueMissing([])
+        self.assertIn('q/[] is missing', str(exc))
+
```

Comment:
Similar story here, you can create a new file called: `bert_e/tests/unit/test_master_queue_missing.py` import `MasterQueueMissing` on top of the file and move those tests there. 

The overall idea is to try to avoid adding unit test into this file that to reduce its size cause it's huge already :D